feat - update ui for sign up, sign in, past call,  settings for logou…

tarakagilefoways opened the pull request:
…t and delete account, cleaned code, added prettier formatter

updated app entry file to support, added ringing call support

softwarebyze commented on .gitignore:
```diff
@@ -31,7 +31,8 @@ yarn-error.*
 .DS_Store
 *.pem
-
+.yarn
```
ive been using bun

tarakagilefoways replied:
in our system we using yarn so i have added that in ignore file

softwarebyze commented on appEntry.js:
```diff
@@ -0,0 +1,20 @@
+setPushMessageHandlers();
+setPushConfig();
+export function appEntry() {
+  const ctx = require.context("./app");
```
this seems unusual. is there any documentation showing something like this?

tarakagilefoways replied:
changed app entry to allow headless activity
similar to below issue, we required to allow few config to run in kill mode
https://github.com/transistorsoft/react-native-background-fetch/issues/493

softwarebyze commented on assets/settings.png:
any reason not to use the expo vector icons? https://icons.expo.fyi/Index/Ionicons/settings-sharp
I think its easier to manage than pngs

tarakagilefoways replied:
changes done verctor icon

softwarebyze commented on app/index.tsx:
```diff
@@ -26,28 +33,53 @@ export default function Index() {
-      <SignIn />
+      <Text style={styles.title}>{"DFIA"}</Text>
+      <Text style={styles.subtitle}>{"Don't Face It Alone"}</Text>
+      <TouchableOpacity style={styles.signInBtn} onPress={onSignIn}>
```
I think these could be Link components to be simpler
https://docs.expo.dev/router/navigating-pages/#buttons

tarakagilefoways replied:
will fix this

softwarebyze commented on app/signup/index.tsx:
```diff
@@ -0,0 +1,243 @@
+      );
+      analytics.track("updateProfile", { username });
+      await updateProfile(userCredential.user, { displayName: username });
+      console.log("User signed in successfully!", userCredential);
```
signed up

tarakagilefoways replied:
will fix this.

tarakagilefoways commented:
@softwarebyze updated code
here is what we have done
1.Replaced all image icon with vector icon, Touchable opacity replaced with link in full app and Removed mentioned log from code :  "user signed in..." to "user signed up" 
2. Filtered past calls for current user only : Done
3.Removed extra console logs from code 
4.Added “oniOSExpoNotificationEvent” method for ios when notification receive, Not able to test because of certificate not received yet
